Pyrado/scripts/evaluation/eval_policies_domain_grid_1d.py

```diff
@@ -131,10 +143,10 @@
             "",
         ]
 
-    elif args.env_name == QQubeSwingUpSim.name:
-        env = QQubeSwingUpSim(dt=args.dt, max_steps=args.max_steps)
+    elif env_name == QQubeSwingUpSim.name:
+        env = QQubeSwingUpSim(dt=dt, max_steps=args.max_steps)
 
-        # param_spec['g'] = np.linspace(9.81*0.7, 9.81*1.3, num=11, endpoint=True)
+        param_spec["g"] = np.linspace(9.81 * 0.7, 9.81 * 1.3, num=11, endpoint=True)
         # param_spec['Rm'] = np.linspace(8.4*0.7, 8.4*1.3, num=11, endpoint=True)
```

Comment:
also run black on the rest (comment out for that)

Reply:
Done with 275179c.